rules.md

```diff
@@ -425,274 +284,367 @@ The Meta % is an added commission that an Admin earns for processing a task clai
 
-Most tasks in Projects, Services and Jobs should require related QA tasks, which should be created at the time of the Work Task, so that there is a clear metric of what the Work task needs to achieve, and how the QA (and the Admin that approves the QA’s claim) can validate that.
+### 4.4.1 Commission Bonuses
 
-QA Tasks must reference the relevant task (i.e. one of the other checklists on the card) and produce some QA report output.
+Each individual Admin's commission is variable based on the number of [Task Claims](#26-claims) they Approved in the previous 30 days (including "today"). The commissions that Admins can earn are tiered as follows:
 
-**Format:**
+<table>
+  <tr>
+   <td><strong># of Claims Approved</strong>
+   </td>
+   <td><strong>Commission Rate</strong>
+   </td>
+  </tr>
+  <tr>
+    <td>0 - 14
+    </td>
+    <td>10%
+    </td>
+  </tr>
+  <tr>
+    <td>15 - 29
+    </td>
+    <td>12%
+    </td>
+  </tr>
+  <tr>
+    <td>30 +
+    </td>
+    <td>15%
+    </td>
+  </tr>
+</table>  
 
-```[qaTaskId])[checklistName]([taskIds])([amount] Dash)```
+Each Commission Bonus is based on the Tier that the Admin’s previous 30 days contributions fall into. The "next" tier of commission is effective immediately once an Admin crosses the threshold, and is effective for all claims as long as the Admin stays within the Tier.
 
 For example:
 
-```1) QA: Production Tasks (1,2) using [spec section 5]([url]) (0.1 Dash)```
+- Admin Alice has processed 10 claims in the past 30 days (including today). If Alice processes 5 additional claims today, Alice will be eligible for a 12% commission on the 5th claim (and any additional claims).
+- Admin Bob has processed 34 claims in the past 30 days, but 7 of those were processed exactly 30 days ago. If Bob processes claims today, each of those will earn 15%. If Bob does not process at least 2 claims today, his commission rate will be 12% for claims processed tomorrow until his 30 day count is 30 or more again, at which point Bob will return to a 15% commission.
+- Admin Carol has processed 9 claims in the past 30 days. Today Carol processes 21 claims. For the first 5 claims Carol processes today the commission is 10%. For the next 15 (claims 15 - 29) the commissions is 12%. The final claim Carol processes has a 15% commission.
+
+## 4.5 Governance
 
-QA Rewards are set as a % added to the relevant Task the QA refers to, defined in the [Price List](#25-price-list).
+Admins are responsible for Governance of the Incubator which is implemented via appointing or removing Admins, or updating these Rules.
 
-QA Tasks claims should be requested by users and processed by Admins like regular Production tasks.  In cases where the QA provider is the Primary Admin, the Secondary Admin is required to approve the claim as with all tasks.
+An Admin Quorum is required in both cases, which means that >= 51% agree (or <=49% don't object) to Admin appointments / removals or changes to the Rules.
```

Comment:
I think "> 50%" would be both easier to understand and more accurate than ">= 51%".  Also might be wise to explicitly state that non-communication means agreement, which is what this implies.

Reply:
done